tests/telemetry/test_telemetry.py

```diff
@@ -31,6 +32,12 @@
 def load_new_cfg(duthost, data):
     duthost.copy(content=json.dumps(data, indent=4), dest=CFG_DB_PATH)
     config_reload(duthost, config_source='config_db', safe_reload=True)
+    # config reload overrides testing telemetry config, ensure testing config exists
+    env = GNMIEnvironment(duthost, GNMIEnvironment.TELEMETRY_MODE)
+    setup_telemetry_forpyclient(duthost)
```

Comment:
>setup_telemetry_forpyclient [](http://example.com/codeflow?start=4&length=27)

Is it better to wait inside this function?

Reply:
Yes, I can make that change. Makes sense since we are restarting telemetry/gnmi inside that function, better for function to return after making sure telemetry is stable.

Reply:
Done